spec/features/merchants/invoices/show_spec.rb

```diff
@@ -83,8 +83,73 @@
       expect(page).to have_content(item_1.name)
       expect(page).to have_content(invoice_item_1.quantity)
       expect(page).to have_content(invoice_item_1.unit_price)
-      expect(page).to have_content(invoice_item_1.status)
+      expect(page).to have_select('invoice_item[status]', selected: "Packaged")
       expect(page).to_not have_content(item_5.name)
     end
+
+    it 'displays the total revenue generated from all of my items on the invoice' do
+      merchant = Merchant.create!(name: 'Schroeder-Jerde')
+      merchant_2 = Merchant.create!(name: 'James Bond')
+      
+      customer_1 = Customer.create!(first_name: 'Sally', last_name: 'Shopper')
+      customer_2 = Customer.create!(first_name: 'Du', last_name: 'North')
+
+      # my items
+      item_1 = merchant.items.create!(name: 'Gold Ring', description: 'Jewelery', unit_price: 10000)
+      item_4 = merchant.items.create!(name: 'Hair Clip', description: 'Accessories', unit_price: 200)
+      item_2 = merchant.items.create!(name: 'Silver Ring', description: 'Jewelery', unit_price: 5000)
+      item_3 = merchant.items.create!(name: 'Hoop Earrings', description: 'Jewelery', unit_price: 1000)
+
+      # other merchant items
+      item_5 = merchant_2.items.create!(name: 'Silver Bracelet', description: 'Accessories', unit_price: 3000)
+      item_6 = merchant_2.items.create!(name: 'Bronze Ring', description: 'Jewelery', unit_price: 2000)
+
+      invoice_1 = customer_1.invoices.create!(status: 1, created_at: "2012-03-06 14:54:15 UTC")
+      invoice_2 = customer_2.invoices.create!(status: 1, created_at: "2012-03-09 14:54:15 UTC")
+      
+      # items for invoice 1
+      invoice_item_1 = InvoiceItem.create!(quantity: 2, unit_price: 10000, item_id: item_1.id, invoice_id: invoice_1.id, status: 1) # $200
+      invoice_item_2 = InvoiceItem.create!(quantity: 2, unit_price: 5000, item_id: item_2.id, invoice_id: invoice_1.id, status: 1) # $100
+      invoice_item_3 = InvoiceItem.create!(quantity: 2, unit_price: 1000,item_id: item_3.id, invoice_id: invoice_1.id, status: 1) # $20
+      invoice_item_5 = InvoiceItem.create!(quantity: 2, unit_price: 3000,item_id: item_5.id, invoice_id: invoice_1.id, status: 1) # Other merchant rev
+      invoice_item_6 = InvoiceItem.create!(quantity: 2, unit_price: 2000,item_id: item_6.id, invoice_id: invoice_1.id, status: 2) # Other merchant rev
```

Comment:
Great job with this test & making sure that the revenue doesn't pull from the 2nd merchant!